[2025-09-01T16:50:51.178+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sap_elt_pipeline.run_dbt_models manual__2025-09-01T16:50:21.574519+00:00 [queued]>
[2025-09-01T16:50:51.188+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sap_elt_pipeline.run_dbt_models manual__2025-09-01T16:50:21.574519+00:00 [queued]>
[2025-09-01T16:50:51.189+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-09-01T16:50:51.219+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): run_dbt_models> on 2025-09-01 16:50:21.574519+00:00
[2025-09-01T16:50:51.225+0000] {standard_task_runner.py:60} INFO - Started process 1932 to run task
[2025-09-01T16:50:51.229+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sap_elt_pipeline', 'run_dbt_models', 'manual__2025-09-01T16:50:21.574519+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/sap_etl_pipeline.py', '--cfg-path', '/tmp/tmpg92uooc6']
[2025-09-01T16:50:51.232+0000] {standard_task_runner.py:88} INFO - Job 18: Subtask run_dbt_models
[2025-09-01T16:50:51.285+0000] {task_command.py:423} INFO - Running <TaskInstance: sap_elt_pipeline.run_dbt_models manual__2025-09-01T16:50:21.574519+00:00 [running]> on host 3c0baf121e58
[2025-09-01T16:50:51.379+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sap_elt_pipeline' AIRFLOW_CTX_TASK_ID='run_dbt_models' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T16:50:21.574519+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-01T16:50:21.574519+00:00'
[2025-09-01T16:50:51.382+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-01T16:50:51.383+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'dbt run --project-dir /opt/***/dbt --profiles-dir /opt/***/dbt']
[2025-09-01T16:50:51.398+0000] {subprocess.py:86} INFO - Output:
[2025-09-01T16:50:53.464+0000] {subprocess.py:93} INFO - [0m16:50:53  Running with dbt=1.9.0-b2
[2025-09-01T16:50:53.466+0000] {subprocess.py:93} INFO - [0m16:50:53  [[33mWARNING[0m]: Deprecated functionality
[2025-09-01T16:50:53.466+0000] {subprocess.py:93} INFO - 
[2025-09-01T16:50:53.467+0000] {subprocess.py:93} INFO - User config should be moved from the 'config' key in profiles.yml to the 'flags' key in dbt_project.yml.
[2025-09-01T16:50:53.470+0000] {subprocess.py:93} INFO - [0m16:50:53  Encountered an error:
[2025-09-01T16:50:53.470+0000] {subprocess.py:93} INFO - Runtime Error
[2025-09-01T16:50:53.471+0000] {subprocess.py:93} INFO -   dbt_project.yml does not parse to a dictionary
[2025-09-01T16:50:53.701+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2025-09-01T16:50:53.714+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2025-09-01T16:50:53.718+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=sap_elt_pipeline, task_id=run_dbt_models, execution_date=20250901T165021, start_date=20250901T165051, end_date=20250901T165053
[2025-09-01T16:50:53.744+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 18 for task run_dbt_models (Bash command failed. The command returned a non-zero exit code 2.; 1932)
[2025-09-01T16:50:53.772+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-09-01T16:50:53.800+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
